jest.setup.tsx

```diff
@@ -1,10 +1,18 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import "regenerator-runtime/runtime";
-import "@testing-library/jest-dom/extend-expect";
+import "@testing-library/jest-dom";
+import "intersection-observer";
+import ResizeObserver from "resize-observer-polyfill";
```

Comment:
Do we need this now we don't use `iframe-resizer`?

Reply:
This is for tests in Jest, because these observers are not available in the Node environment. When the removal of `iframe-resizer` is released, I'll check if it's needed in the tests or if it can be removed.

Reply:
Added a comment to clarify why this is needed.